.github/workflows/publishRelease.yaml

```diff
@@ -0,0 +1,134 @@
+        with:
+          versionString: "${{ github.event.head_commit.message }}"
+      - name: workflow-setup
+        id: workflow-setup
+        uses: hyperledger/indy-shared-gha/.github/actions/workflow-setup@main
+          
+  createRelease:
+    name: Create Release
+    needs: [release-infos]
+    if: needs.release-infos.outputs.isVersionBump == 'true'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        
+      - name: Download Plenum Artifacts from Github Action Artifacts
+        uses: dawidd6/action-download-artifact@v2
+        with:
+          github_token: ${{secrets.GITHUB_TOKEN}}
+          workflow: releasepr.yaml
+          workflow_conclusion: success
+          name: plenum-deb
+          path: artifacts/plenum-deb
+      - name: Download Plenum Artifacts from Github Action Artifacts
+        uses: dawidd6/action-download-artifact@v2
+        with:
+          github_token: ${{secrets.GITHUB_TOKEN}}
+          workflow: releasepr.yaml
+          workflow_conclusion: success
+          name: plenum-python
+          path: artifacts/plenum-python
+      - name: Download Plenum Artifacts from Github Action Artifacts
+        uses: dawidd6/action-download-artifact@v2
+        with:
+          github_token: ${{secrets.GITHUB_TOKEN}}
+          workflow: releasepr.yaml
+          workflow_conclusion: success
+          name: third-party-dependencies
+          path: artifacts/third-party-dependencies
+      - uses: actions/upload-artifact@v2
+        with:
+          name: third-party-dependencies
+          path: artifacts/third-party-dependencies
+          retention-days: 5
+      - uses: actions/upload-artifact@v2
+        with:
+          name: plenum-deb
+          path: artifacts/plenum-deb
+          retention-days: 5
+      - uses: actions/upload-artifact@v2
+        with:
+          name: plenum-python
+          path: artifacts/plenum-python
+          retention-days: 5
+      - name: Zip Files for Release
+        run: |
+          zip -r artifacts/plenum-deb.zip artifacts/plenum-deb
+          zip -r artifacts/plenum-python.zip artifacts/plenum-python
+          zip -r artifacts/third-party-dependencies.zip artifacts/third-party-dependencies
+      - name: Release
+        uses: softprops/action-gh-release@v1
+        with:
+          tag_name: ${{ needs.release-infos.outputs.VERSIONTAG }}
+          files: |
+            artifacts/**.zip
+          generate_release_notes: true
+          body: "[${{ needs.release-infos.outputs.VERSIONTAG }}] "
+          prerelease: ${{ needs.release-infos.outputs.isPreRelease }}
+          target_commitish: ${{github.event.ref}}
+          name: "${{ needs.release-infos.outputs.VERSIONTAG }}"
+          token: ${{ secrets.BOT_PR_PAT }}
+
+  publish_artifacts:
+    name: Publish Artifacts
+    needs: [release-infos, createRelease]
+    if: needs.release-infos.outputs.isVersionBump == 'true'
```

Comment:
This job should also be protect against publishing when running in a different fork.  The publish output will need to be setup in `release-infos` and then used here.